flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/typeutils/SerializableAvroSchema.java

```diff
@@ -21,13 +21,13 @@
 import org.apache.flink.annotation.Internal;
 
 import org.apache.avro.Schema;
-import org.apache.avro.Schema.Parser;
```

Comment:
nit: why remove the import?

Reply:
IDE's work that forces "Parser" -> "Schema.Parser". I'll manually tweak it back.